### ISSUE TITLE
Extension freezes Burp on start

### ISSUE THREAD

**Issue by denevin** [augustd/burp-suite-software-version-checks#97]
On Windows 10, when I close Burp program after working on a project, then start it again after few minutes with default settings, quite often the program will freeze.
I narrowed it down to this happening when either “Error Message Checks” or “Software Version Reporter” extension is enabled.
I’m not sure about other extensions.
It’s possible to reproduce this issue quickly on my computer after starting and stopping burp about 3-4 times.

Also when I start Burp I usually click on the maximize button when it’s loading before it starts, after which it resizes when it loads. 
However I am able to reproduce this issue without having to click resize button on start.

**Comment by mavensecurity** [comment]
Agreed.  "Software Vulnerability Reporter" (as Burp calls it) causes Burp Suite Pro to freeze on Windows 10 when loading prior project where this extension was loaded.  The only work around is having to remember to disable the extension before closing the project, then re-enable after resuming project later.  Alternatively resume project with all extensions disable, and turn on the ones you want.
